cmake/modules/SwiftSupport.cmake

```diff
@@ -106,11 +110,22 @@ function(add_swift_target target)
                          ${source}
                          ${AST_DEPENDS}
                        COMMAND
-                         ${CMAKE_SWIFT_COMPILER} -frontend ${compile_flags} -emit-module-path ${mod} -emit-module-doc-path ${doc} -o ${obj} -c ${all_sources})
+                         ${CMAKE_SWIFT_COMPILER} -frontend ${compile_flags} -emit-module-path ${mod} -emit-module-doc-path ${doc} -o ${obj} -c "@${resp_file}")
 
     list(APPEND objs ${obj})
+    set(objs_resp_file ${CMAKE_CURRENT_BINARY_DIR}/${target}.dir/objects.resp)
+    list(JOIN objs " " objs_resp_text)
+    file(WRITE ${objs_resp_file} ${objs_resp_text})
+
     list(APPEND mods ${mod})
+    set(mods_resp_file ${CMAKE_CURRENT_BINARY_DIR}/${target}.dir/modects.resp)
```

Comment:
Similar.  Also, did you mean modsects?  Whats the `ects` suffix supposed to stand for?

Reply:
eee, should just be mods, bad search and replace after copying it from the objects one.